### DISCUSSION TITLE
[21.02]发现了一些挺有意思的问题…

### DISCUSSION THREAD

**Discussion by jh0072110** [immortalwrt/immortalwrt#635] [General]
我发现在不同浏览器中，中文会出现没翻译的情况
iOS Safari：
![IMG_3B57FEC81E78-1](https://user-images.githubusercontent.com/50496854/158929689-2276fecf-82ff-4d31-a110-43fd76c9a74b.jpeg)
Desktop Firefox and Chrome：
<img width="356" alt="image" src="https://user-images.githubusercontent.com/50496854/158929752-1bb7d907-9622-498b-ada7-379860b6cb1f.png">
PS：其实Firefox在auto还会默认英文，需要把auto强制成简体中文才可以
挺神奇的不知道算不算bug的问题，放在discussions里研究研究哈哈哈哈

**Comment by 1715173329** [comment]
> 我发现在不同浏览器中，中文会出现没翻译的情况

这个应该是 js LuCI 的 feature 了，自从它出现以来就一直有这样的现象

> 其实Firefox在auto还会默认英文，需要把auto强制成简体中文才可以

应该是通过 `accept-language` 这个 header 来判断使用者语言的，你可以检查一下。

**Reply by jh0072110** [reply]
哈哈哈哈谢谢大佬解惑

**Reply by thisdk** [reply]
遇到这种情况 Ctrl + F5 强制刷新.就OK了.
